README.md

```diff
@@ -1,2 +1,161 @@
+
+    private val yotiDocScan = YotiDocScan(this)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+
+        startButton.setOnClickListener {
+            yotiDocScan
+                    .setSessionId("<Your Session ID>")
+                    .setSessionToken("<Your Session Token>")
+                    .start(this)
+        }
+
+    }
+}
+```
+
+## Retrieve status of the session
+
+Once the user completed the flow in the SDK, the user will be redirected to the Activity which started the SDK.
+You can then retreive the current status of the current session like that:
+
+```kotlin
+class MainActivity : AppCompatActivity() {
+
+    private val yotiDocScan = YotiDocScan(this)
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (requestCode == YOTI_DOCS_REQUEST_CODE) {
+
+            val sessionStatusCode = yotiDocScan.sessionStatusCode
+            val sessionStatusDescription = yotiDocScan.sessionStatusDescription
+
+            result_text.text = "Result code: $sessionStatusCode Message: $sessionStatusDescription"
+        }
+    }
+}
+```
+
+
+### Possible status for the session
+
+| Code              | Message                      | Retry possible for the same session                    |
+| ----------------- | ---------------------------- | ---------------------------------- |
+| 0                 | Result with success          | No                                 |
+| 1000              | No error occurred - the end-user cancelled the session for an unknown reason.          | Yes |
+| 2000              | Unauthorised request (wrong or expired session token).          | Yes |
+| 2001              | Session not found.          | Yes |
+| 2003              | SDK launched without session Token.          | Yes |
+| 2004              | SDK launched without session ID.          | Yes |
+| 3000              | Yoti's services are down or unable to process the request.          | Yes |
+| 3001              | An error occurred during a network request          | Yes |
+| 3002              | User has no network          | Yes |
+| 4000              | The user did not grant permissions to the camera          | Yes |
+| 5000              | No camera.(When user's camera was not found and file upload is not allowed)          | No |
+| 6000              | SDK is out-of-date - please update the SDK to the latest version          | No |
+| 6001              | Unexpected internal error          | No |
+| 6002              | Unexpected document scanning error          | No |
+
+
+## Customisation
+You can customise the appearance of the screens of the SDK by overriding some styles.
+
+### Font Family and Font Colour
+In order to change the font family and the font colour you just need to define the following styles:
```

Comment:
This fonts are part of the Yoti Android UI styles which are used by the SDK, the SDK ones are: 
YotiDocs.TextView.Title
YotiDocs.TextView.Header
YotiDocs.TextView.Text
YotiDocs.TextView.Link